### ISSUE TITLE
The table ui will break if long name

### ISSUE THREAD

**Issue by abawchen** [Parallels/rq-dashboard#168]
Hi,

Thanks for the great library :)

I found the table ui will break if long name:

![screen shot 2019-01-02 at 5 50 00 pm](https://user-images.githubusercontent.com/2952415/50586970-dd3b5280-0eb6-11e9-8a14-4bc8c75d69d7.png)

By adding following css would help
```css
table {
    table-layout: fixed;
    word-wrap: break-word;
}
```

![screen shot 2019-01-02 at 5 48 11 pm](https://user-images.githubusercontent.com/2952415/50587004-06f47980-0eb7-11e9-9c72-b4262bd06cd5.png)

Make sense? If yes, I would like to create a pull request.

**Comment by eoranged** [comment]
Yes. Nice idea, @abawchen !
Feel free to create a new pull request.

**Comment by erikvanzijst** [comment]
Though it might be nice if the columns resize to minimize the amount of wrapping as long as there's still space left in the other two columns.

**Comment by eoranged** [comment]
Fixed in #171, closing.
Feel free to reopen or create additional issue for additional improvements in long names handling.
